iocore/net/QUICNetVConnection.cc

```diff
@@ -94,8 +95,67 @@ QUICNetVConnection::do_io_write(Continuation *c, int64_t nbytes, IOBufferReader
 }
 
 int
-QUICNetVConnection::startEvent(int /*event ATS_UNUSED */, Event *e)
+QUICNetVConnection::acceptEvent(int event, Event *e)
 {
+  EThread *t    = (e == nullptr) ? this_ethread() : e->ethread;
+  NetHandler *h = get_NetHandler(t);
+
+  MUTEX_TRY_LOCK(lock, h->mutex, t);
+  if (!lock.is_locked()) {
+    if (event == EVENT_NONE) {
+      t->schedule_in(this, HRTIME_MSECONDS(net_retry_delay));
+      return EVENT_DONE;
+    } else {
+      e->schedule_in(HRTIME_MSECONDS(net_retry_delay));
+      return EVENT_CONT;
+    }
+  }
+
+  thread = t;
+
+  // Send this NetVC to NetHandler and start to polling read & write event.
+  if (h->startIO(this) < 0) {
+    free(t);
+    return EVENT_DONE;
+  }
+  this->read.enabled = 1;
```

Comment:
This is a hack now, because no one calls do_io_xxx of QUICNetVC.

Can you add a comment here, so we can complete it later.